draft-ietf-cbor-7049bis.md

```diff
@@ -1152,7 +1152,7 @@ help to have some distinguishing characteristics for the data itself.
 
 Tag number 55799 is defined for this purpose.  It does not impart any special
 semantics on the data item that it encloses; that is, the semantics of
-the tag content data item enclosed in tag number 55799 is exactly identical to the semantics
+the tag content enclosed in tag number 55799 is exactly identical to the semantics
 of the data item itself.
 
```

Comment:
But not here!

Reply:
We disagree, but if you really want to revert, go ahead.